content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -451,19 +451,17 @@ customize policies to individual workloads, and secondly, it may be challenging
 single level of "tenancy" that should be given a namespace. For example, an organization may have
 divisions, teams, and subteams - which should be assigned a namespace?
 
-To solve this, Kubernetes provides the [Hierarchical Namespace Controller (HNC)](https://github.com/kubernetes-sigs/hierarchical-namespaces),
-which allows you to organize your namespaces into hierarchies, and share certain policies and
-resources between them. It also helps you manage namespace labels, namespace lifecycles, and
-delegated management, and share resource quotas across related namespaces. These capabilities can
-be useful in both multi-team and multi-customer scenarios.
-
-Other projects that provide similar capabilities and aid in managing namespaced resources are
-listed below.
+To solve this, you could use a third-party project listed below which allows you to organize your
+namespaces into hierarchies, and share certain policies and resources between them. These can also
+help you manage namespace labels, namespace lifecycles, and delegated management, and share resource
+quotas across related namespaces. These capabilities can be useful in both multi-team and
+multi-customer scenarios.
 
 #### Multi-team tenancy
 
 {{% thirdparty-content %}}
 
+* [Accurate](https://github.com/cybozu-go/accurate)
```

Comment:
let's separate out adding new third-party links from the HNC change ... I'm not quite sure the rationale used to select which links to include here

Reply:
If none of these links are allowed, we should just remove all mention of namespace hierarchy, right? This PR would be pointless in anything resembling its current form if that's the case.

Reply:
In https://github.com/kubernetes/website/pull/50519/commits/1eeb584c887f41ce28a39f59a8d351c432c838db I brought this up to the standard documented in your references.

Reply:
I can't in good conscience remove the HNC reference while leaving its most direct successor from the list of alternative implementations, hoping that the inconsistency eventually gets reconciled.

I think we're at an impasse with this suggestion, so I'm closing the PR.

Thanks for your review :)

Reply:
ack, thanks for jumping in to help with the tidying anyway

opened https://github.com/kubernetes/website/pull/50544 with my suggestion at https://github.com/kubernetes/website/pull/50519#discussion_r2050665719

will ping #sig-multicluster folks to follow-up on https://github.com/kubernetes/website/pull/50519#discussion_r2050971738

Reply:
Hello,

I am a maintainer of accurate, the mentioned project.
I can say accurate resembles HNC quite much and has been long maintained by Cybozu, one of the CNCF members.

Please let me know if I can help with the discussion about including third-party links.

Reply:
While a multi-tenant operator isn't strictly required to run Kubernetes, it is a blocker for any user with the desire to share a Kubernetes cluster between multiple tenants. And it's generally more user-friendly to include links to alternatives from the Kubernetes documentation. I'll vote for raising a dedicated PR to add Accurate to the existing list. (And probably also update the link to Capsule, which is https://github.com/projectcapsule/capsule now.)